packages/frontend/src/components/Strategies/Crab/CrabTradeV2.tsx

```diff
@@ -146,6 +168,50 @@ const CrabTradeV2: React.FC<CrabTradeV2Type> = ({ maxCap, depositedAmount }) =>
     setStrategyData()
   }, [])
 
+const { depositPriceImpactWarning, withdrawPriceImpactWarning } = useMemo(() => {
+  let depositPriceImpactWarning: Boolean | false
+  let withdrawPriceImpactWarning: Boolean | false
+
+  const squeethPrice = depositOption === 0 ? ethAmountOutFromDeposit.div(squeethAmountInFromDeposit)
+                                           : ethAmountInFromWithdraw.div(squeethAmountOutFromWithdraw)
+  const scalingFactor = new BigNumber(10000)
+  const fundingPeriod = new BigNumber(17.5).div(365)  
+  const executionVol = new BigNumber(Math.log(scalingFactor.times(squeethPrice).div(normFactor.times(ethIndexPrice)).toNumber())).div(fundingPeriod).sqrt()
+  const showPriceImpactWarning = (executionVol.minus(impliedVol)).abs().gt(BigNumber.max(new BigNumber(impliedVol).div(10), 0.08))
```

Comment:
Probably best to use constants for the 10% and the 8%, so can be changed easily in future.